topostats/run_topostats.py

```diff
@@ -176,16 +176,16 @@ def process_scan(
     dna_traces = dnaTrace(
         full_image_data=grains.images["gaussian_filtered"].T,
         grains=grains.images["labelled_regions"],
-        afm_image_name=filtered_image.filename,
+        filename=filtered_image.filename,
         pixel_size=filtered_image.pixel_to_nm_scaling,
-        number_of_columns=grains.images["labelled_regions"].shape[0],
-        number_of_rows=grains.images["labelled_regions"].shape[1],
     )
+    dna_traces.trace_dna()
```

Comment:
Sorry if I've asked this before, but should it be a configurable option to run this? Maybe a future decsision.

Reply:
Its a good point and we should look at making the end-point of processing optional, I've documented it as an issue to be addressed (see #185 ).